tests/subsys/settings/fcb_init/src/settings_test_fcb_init.c

```diff
@@ -26,8 +26,10 @@ static u32_t val32;
 /* leverage that this area has to be embededd flash part */
 #ifdef DT_FLASH_AREA_IMAGE_0_ID
 static const volatile __attribute__((section(".rodata")))
-__aligned(DT_FLASH_WRITE_BLOCK_SIZE)
-u8_t prepared_mark[DT_FLASH_WRITE_BLOCK_SIZE] = {ERASED_VAL};
+#define FLASH_WRITE_BLOCK_SIZE \
```

Comment:
This macro is showing up right in the middle of the declaration; can you move it up above the `static const volatile ...` line?

Reply:
Fixed.